src/clj/rems/db/csv.clj

```diff
@@ -0,0 +1,115 @@
+(ns rems.db.csv
+  "Utilities for exporting database contents as CSV"
+  (:require [clj-time.core :as time]
+            [clojure.string :as str]
+            [rems.config :refer [env]]
+            [rems.db.user-settings :as user-settings]
+            [rems.text :as text]))
+
+(defn print-to-csv [& {:keys [column-names rows
+                              enclose-strings-in-quotes separator]
```

Comment:
could just call `enclose-strings-in-quotes` something shorter like `quote?` or `quote-strings?`

Reply:
Done.